### DISCUSSION TITLE
Update 6.10 - dev.1

### DISCUSSION THREAD

**Discussion by TBog** [TBog/TBLauncher#260] [Pre-Release]
* add "show search desktop with keyboard" gesture
* add more result list preferences
* merge color and opacity preferences
* add migration for preferences
* fix "Reset preferences" crash
* add the settings activity to the activity stack (now you can more easily test preferences)
* fix "Show Dock" preference and move "dock visible for results" to the search desktop

<hr /><em>This discussion was created from the release <a href='https://github.com/TBog/TBLauncher/releases/tag/v6.10-dev.1'>Update 6.10 - dev.1</a>.</em>

**Comment by ltguillaume** [comment]
1. I believe since [Update 6.9 - dev.3](https://github.com/TBog/TBLauncher/releases/tag/v6.9-dev.3), the dock suddenly had completely different margins (narrower)
2. Now the result list has a custom horizontal & vertical margin setting.

=> Would you consider adding the same 2 margin settings for the dock, so that I can revert the changes?

**Reply by TBog** [reply]
I was going to add the margin preferences to both search bar and dock.

I personally don't use those options and I wanted to try out the feature before I implement it everywhere.

![my screen](https://user-images.githubusercontent.com/6465804/156912426-444b968c-7683-4c7e-a50c-55e9ed93dcf0.jpg)

**Reply by ltguillaume** [reply]
Yeah, would be great to have the same parameters everywhere! My set-up used to look like this, with the dock aligned with the results:
<img src="https://user-images.githubusercontent.com/4777345/156926800-1d7eecad-fda9-46f7-bda8-1ea5b2cfc306.png" height="800">

As you can see, there weren't many items available in the result list, so the vertical margin between items, so it would maybe be possible to change the "vertical margin" parameter to apply to the _individual result list items_, so you can adjust the spacing.

It already was difficult to get the icon size the same, but if the dock and search bar also get the parameters icon size, horizontal and vertical margin, then it's a lot easier and logical 🙂

**Reply by TBog** [reply]
The dock icon size is computed according to the amount of items and bar height.
